index.js

```diff
@@ -10,6 +10,7 @@
 
     each(tabs, function(i, tab) {
       tab.addEventListener('click', function(e) {
+        e.preventDefault();
```

Comment:
Should add an option for users to choose whether they want to preventDefault or not